.pre-commit-config.yaml

```diff
@@ -8,3 +8,12 @@ repos:
     rev: "v2.7.1"
     hooks:
       - id: prettier
+  - repo: https://github.com/python-poetry/poetry
+    rev: "1.5.1"
+    hooks:
+      - id: poetry-check
+      - id: poetry-lock
+      - id: poetry-export
+        args: ["-f", "requirements.txt", "-o", "requirements.txt"]
+      - id: poetry-export
+        args: ["--dev", "-f", "requirements.txt", "-o", "requirements-dev.txt"]
```

Comment:
With the transition to a `pyproject.toml` file and Poetry these requirements files might not be necessary. Consider removing these lines and the `requirements*.txt` files to help simplify and unify setup maintenance or implementation. If they are still needed as a backup or for compatibility of some kind, consider adding comments here and within the docs to help describe this.

Reply:
Done.

Reply:
I also removed the poetry lock pre-commit hook because I think it would be best if the `poetry.lock` file only changes after making changes in response to changes to the `pyproject.toml` file.